pkg/util/process_file_stats_linux.go

```diff
@@ -0,0 +1,41 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux
+// +build linux
+
+package util
+
+import (
+	"os"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/shirou/gopsutil/v3/process"
+)
+
+// getProcessFileStats returns the number of file handles the Agent process has open
```

Comment:
linter will complain here, since `GetProcessFileStats` is public, it should have a comment documenting it.
```suggestion
// GetProcessFileStats returns the number of file handles the Agent process has open
```